src/tribler/core/components/bandwidth_accounting/db/database.py

```diff
@@ -114,7 +120,7 @@ def get_total_taken(self, public_key: bytes) -> int:
         :param public_key: The public key of the peer of which we want to determine the total taken.
         :return The total amount of bandwidth taken by the specified peer, in bytes.
         """
-        return sum(transaction.amount for transaction in self.BandwidthTransaction
+        return orm.sum(transaction.amount for transaction in self.BandwidthTransaction
```

Comment:
**nitpick:** It is also possible to use `sum` method of a query instead of the `sum` function to avoid `orm` import:
`return select(transaction.amount for transaction in ...).sum()`